test/slugify-filter/apostrophe.njk

```diff
@@ -0,0 +1,5 @@
+---
+title: "Hi I'm Zach"
```

Comment:
Not sure you need this test case since the `comma.njk` filter test below has a comma and apostrophe.

Reply:
1. I think you'd also want tests which pass custom options to the `slug` and `slugify` filters (since thats a new feature)
2. Possibly a couple tests for Liquid engine, since that seems to be slightly different than Nunjucks implementation for passing complex args as options (based on a quick test).